importer/main.py

```diff
@@ -1110,8 +1183,15 @@ def encode_image(image_file):
 # and saves it as a Binary resource. It returns the id of the Binary resource if
 # successful and 0 if failed
 def save_image(image_source_url):
-    headers = {"Authorization": "Bearer " + config.product_access_token}
+    try:
+        headers = {"Authorization": "Bearer " + config.product_access_token}
```

Comment:
@Wambere why are we using a separate access token here? I think `config.access_token` should work

Reply:
@dubdabasoduba the assumption is that it is possible to have the images coming from a source that requires different authentication? I think it's better to have the option and just copy the same token over than not, no?